eks/variables.tf

```diff
@@ -34,59 +36,26 @@ variable "eks_access_entry_map" {
       }), {}) # access_scope
     })), {})  # access_policy_associations
   }))
-  default = {}
+  description = "Represents a map of access entries for an EKS cluster. Each entry in the map represents the access configuration for a specific principal ARN"
+  default     = {}
 }
 
 variable "kubernetes_version" {
-  type    = string
-  default = "1.26"
+  type        = string
+  description = "The desired Kubernetes master version. If you do not specify a value, the latest available version is used."
+  default     = "1.26"
```

Comment:
I picked up this version since I recall you were locally running FF on this version without issues.
My approach here is to change the default value once we confirm internally that there are no issues with FF on specific Kubernetes version.

Reply:
@hardillb in fact, if we can somehow confirm FF compatibility with a higher Kubernetes version then I am fine with setting a different default version.

Reply:
The test "scaling" cluster is currently running 1.26 and I think that is newest we have actually tested recently.

My cluster is running 1.24 and needs upgrading. Leave this as it is for now, but we should open a ticket in CloudProject to upgrade the scaling cluster as a test and then update.